hero/karin.json

```
@@ -0,0 +1,290 @@
+                {
+                    "description": "+5% damage dealt",
+                    "resources": [{ "item": "molagora", "qty": 1 }, { "item": "gold", "qty": 4000 }]
+                },
+                {
+                    "description": "+10% damage dealt",
+                    "resources": [
+                        { "item": "twisted-fang", "qty": 1 },
+                        { "item": "molagora", "qty": 4 },
+                        { "item": "gold", "qty": 21000 }
+                    ]
+                },
+                {
+                    "description": "+10% damage dealt",
+                    "resources": [
+                        { "item": "twisted-fang", "qty": 1 },
+                        { "item": "molagora", "qty": 7 },
+                        { "item": "gold", "qty": 33000 }
+                    ]
+                },
+                {
+                    "description": "+10% damage dealt",
+                    "resources": [
+                        { "item": "blazing-soul", "qty": 2 },
+                        { "item": "molagorago", "qty": 1 },
+                        { "item": "gold", "qty": 80000 }
+                    ]
+                }
+            ],
+            "buffs": [],
+            "debuffs": []
+        },
+        {
+            "isPassive": false,
+            "soulBurn": 0,
+            "selfSkillBarValue": 0,
+            "soulBurnEffect": "",
+            "awakenUpgrade": true,
+            "cooldown": 4,
+            "name": "Blade Art: Flash",
+            "soulAcquire": 2,
+            "description": "Slashes the enemy. A critical hit will decrease Defense for 1 turn and grant an extra turn.",
+            "enhancement": [
+                {
+                    "description": "+5% damage dealt",
+                    "resources": [{ "item": "molagora", "qty": 1 }, { "item": "gold", "qty": 4000 }]
+                },
+                {
+                    "description": "+10% damage dealt",
+                    "resources": [
+                        { "item": "twisted-fang", "qty": 1 },
+                        { "item": "molagora", "qty": 4 },
+                        { "item": "gold", "qty": 21000 }
+                    ]
+                },
+                {
+                    "description": "+10% damage dealt",
+                    "resources": [
+                        { "item": "twisted-fang", "qty": 1 },
+                        { "item": "molagora", "qty": 7 },
+                        { "item": "gold", "qty": 33000 }
+                    ]
+                },
+                {
+                    "description": "+10% damage dealt",
+                    "resources": [
+                        { "item": "blazing-soul", "qty": 2 },
+                        { "item": "molagorago", "qty": 1 },
+                        { "item": "gold", "qty": 80000 }
+                    ]
+                }
+            ],
+            "buffs": [],
+            "debuffs": []

```

Skill says on crit, does def down. It is not listed on side of skill, so not sure if it is for caster or enemy.  I don't have Karin to test.

```suggestion
            "debuffs": ["stic_def_dn"]
```

---

Yep, she does def break enemies on crit, good find. Maybe the game doesn't show since is conditional to a crit.. idk, good to add.